README.md

```diff
@@ -3,13 +3,13 @@
 [![Build Status](https://travis-ci.org/Pand-Aid /pandaid-api.svg?branch=master)](https://travis-ci.org/Pand-Aid /pandaid-api)
```

Comment:
Do you mind fixing this link too? I think it's broken as well cause of the space in it.